.github/CONTRIBUTING.md

```diff
@@ -48,9 +50,21 @@ Push the database schema to Planetscale:
 
 > Make sure you replace `user`, `password`, `host` and `db` with your own values
 
+#### Windows
+
+```sh-session
+cd internal\db
+
+set DRIZZLE_DATABASE_URL=mysql://{user}:{password}@{host}/{db}
+pnpm drizzle-kit push:mysql
+set DRIZZLE_DATABASE_URL=
+```
+
+#### Mac/Linux
+
 ```sh-session
 cd internal/db
-DRIZZLE_DATABASE_URL='mysql://{user}:{password}@{host}/{db}?ssl={"rejectUnauthorized":true}' pnpm drizzle-kit push:mysql
```

Comment:
why did you remove it?
last time I checked it was required

Reply:
Oh it is? I thought `ssl` was a client security option.
`?` can cause issues on Windows PowerShell, so I thought best to remove it.
If it's needed I'll have to do a little testing to find a command that works in both cmd and PowerShell.